List Ubuntu 18.04 as being supported (#11)
anbox-cloud/discourse-docs: requirements.md

Supported Ubuntu versions:

* 18.04 (bionic)
* 20.04 (focal)

> For new deployments Ubuntu 20.04 (focal) should be preferred.

Reviewer: ```suggestion
For new deployments, Ubuntu 20.04 (focal) is preferred.
```
